exercises/concept/lasagna/.docs/hints.md

```diff
@@ -4,10 +4,20 @@
 
 - An [integer value][integers] can be defined as one or more consecutive digits.
 
+If you see a `panic:` error when running the tests, this is as you haven't implemented one of the functions (it should say which one) or you have left the boilerplate in place, you need to remove the `panic(...)` line from the supplied code.
+
 ## 1. Define the expected oven time in minutes
 
 - You need to define a [constant][constants] and assign it the expected oven time in minutes.
 
+If you see an `undefined: OvenTime` error then double check that you have the constant defined.
```

Comment:
I wonder if these should be hints?

Reply:
Good point, the spec says these should be bullet points!